.github/workflows/velox-benchmark.yml

```diff
@@ -40,17 +40,40 @@ jobs:
         with:
           path: velox-testing
 
-      - name: Setup Velox Testing and Velox
+      - name: Setup Velox
         uses: ./velox-testing/.github/actions/velox-setup
         with:
           repository: ${{ inputs.repository }}
           velox_commit: ${{ inputs.velox_commit }}
 
-      - name: "Build Velox GPU with Benchmarks (SHA: ${{ env.VELOX_COMMIT_SHA }})"
+      - name: Download and Load Velox Dependencies Container Image
```

Comment:
We seem to have some duplication between this file and velox-test.yml. Can we factor out common steps into a shared file/action?